src/jira/connect.ts

```diff
@@ -78,6 +78,14 @@ export default async (_: Request, res: Response): Promise<void> => {
 					},
 					homeUrl: "https://github.com/features/actions",
 				},
+				jiraRemoteLinkInfoProvider: {
+					key: "github-remotelinks-integration",
+					name: {
+						value: "GitHub Security Reports"
```

Comment:
Needs PM approval

Reply:
After slack discussion and PM chat, changed to "GitHub" now